src/libraries/System.Private.CoreLib/src/System/Runtime/ExceptionServices/ExceptionHandling.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading;
+
+namespace System.Runtime.ExceptionServices
+{
+    public delegate bool UnhandledExceptionHandler(System.Exception exception);
```

Comment:
@VSadov the approved shape used `Func<Exception, bool>` instead of a custom delegate.

Reply:
> @VSadov the [approved API shape](https://github.com/dotnet/runtime/issues/101560#issuecomment-2218393125) used `Func<Exception, bool>` instead of a custom delegate.

@VSadov do you have plans to address this? We cannot check in new unapproved public APIs.

Reply:
Opened #110254.

Reply:
> do you have plans to address this? We cannot check in new unapproved public APIs.

Yes. It was an oversight. 
The approved shape was the same as in proposal except for this part. I missed that there was a diff from the proposal.